nselib/libssh2.luadoc

```diff
@@ -0,0 +1,120 @@
+--
```

Comment:
This needs to be `---` (3 hyphens) in order for this block to be parsed as NSEdoc.